frontend/javascripts/viewer/view/action_bar_view.tsx

```diff
@@ -160,31 +167,67 @@ function CreateAnnotationButton() {
       maybeMappingName = mappingInfo.mappingName;
     }
 
-    const annotation = await createExplorational(
-      dataset.id,
-      "hybrid",
-      false,
-      fallbackLayerName,
-      maybeMappingName,
+    createExplorational(dataset.id, "hybrid", false, fallbackLayerName, maybeMappingName).then(
+      (annotation) => {
+        history.push(`/annotations/${annotation.id}${location.hash}`);
+      },
     );
-    history.push(`/annotations/${annotation.id}${location.hash}`);
+  };
+
+  const getFallbackLayerName = (segmentationLayer: APISegmentationLayer | null | undefined) => {
+    return segmentationLayer && doesSupportVolumeWithFallback(dataset, segmentationLayer)
+      ? segmentationLayer.name
+      : null;
+  };
+
+  const onClick = async () => {
+    // This will be set is cases where it is clear which layer to use.
+    const obviousSegmentationLayer = getUnambiguousSegmentationLayer();
```

Comment:
I'd stick to one term (currently, you use "obvious" and "unambiguous"). I think, I prefer the latter.